model/src/main/java/io/neow3j/model/types/StackItemType.java

```diff
@@ -15,6 +15,17 @@
     MAP("Map", 0x48),
     INTEROP_INTERFACE("InteropInterface", 0x60);
 
+    public static final String ANY_VALUE = "Any";
```

Comment:
hmmmmmm...... isn't it possible to use the Enums? If not, then, use the values from below to the ENUMs value.... like this:

```
ANY(ANY_VALUE, 0x00);
```

If you can't do this, then declare `ANY_VALUE` as `public static final String` somewhere else (like in a file called `StackItemTypeConstants.java`.

Reply:
It's not possible to use the enums, since the Subtypes' name declaration in the abstract class StackItem must be final. I changed it according your suggestion.